examples/customization/README.md

```diff
@@ -39,7 +39,7 @@ The table below summarizes all of the options. For some of them, there are examp
 | N/A | `http-snippets` | Sets a custom snippet in http context. | N/A | |
 | `nginx.org/location-snippets` | `location-snippets` | Sets a custom snippet in location context. | N/A | |
 | `nginx.org/server-snippets` | `server-snippets` | Sets a custom snippet in server context. | N/A | |
-| `nginx.org/lb-method` | `lb-method` | Sets the [load balancing method](https://www.nginx.com/resources/admin-guide/load-balancer/#method). The default `""` specifies the round-robin method. | `""` | |
+| `nginx.org/lb-method` | `lb-method` | Sets the [load balancing method](http://nginx.org/en/docs/http/load_balancing.html#nginx_load_balancing_methods). The default `""` specifies the `least_conn` method. | `""` | |
```

Comment:
Better to use this link -- https://docs.nginx.com/nginx/admin-guide/load-balancer/http-load-balancer/#choosing-a-load-balancing-method
Please mention: To use the round-robin method, specify `round_robin`. Remove "The default `""` specifies the `least_conn` method"
Also, add `least_conn` to the default column.